content/en/faq/distributed-tracing/how-to-support-tracing.md

```diff
@@ -19,4 +19,4 @@ If you are using LightStep, you will also need to forward the following headers:
 
 - `x-ot-span-context`
 
-Header propagation may be accomplished through client libraries, such as [Zipkin](https://zipkin.io/pages/tracers_instrumentation.html) or [Jaeger](https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-core#b3-propagation). It may also be accomplished manually, as documented in the [Distributed Tracing Task](/docs/tasks/observability/distributed-tracing/overview#trace-context-propagation).
\ No newline at end of file
+Header propagation may be accomplished through client libraries, such as [Zipkin](https://zipkin.io/pages/tracers_instrumentation.html) or [Jaeger](https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-core#b3-propagation). It may also be accomplished manually, as documented in the [Distributed Tracing Task](/docs/tasks/observability/distributed-tracing/overview#trace-context-propagation).
```

Comment:
This is strange, I don't see anything changed on this line. Why is it showing a diff?

Reply:
Probably because of the newline character.